tidy3d/components/monitor.py

```diff
@@ -319,6 +319,28 @@ def storage_size(self, num_cells: int, tmesh: Array) -> int:
         return BYTES_REAL * num_steps * num_cells * len(self.fields)
 
 
+class PermittivityMonitor(FreqMonitor):
+    """:class:`Monitor` that records the diagonal components of the complex-valued relative
+    permittivity tensor in the frequency domain. The recorded data has the same shape as a
+    :class:`.FieldMonitor` of the same geometry: the permittivity values are saved at the
+    Yee grid locations, and can be interpolated to any point inside the monitor.
+
+    Example
+    -------
+    >>> monitor = PermittivityMonitor(
+    ...     center=(1,2,3),
+    ...     size=(2,2,2),
+    ...     freqs=[250e12, 300e12],
+    ...     name='eps_monitor')
+    """
+
+    _data_type: Literal["PermittivityData"] = pydantic.Field("PermittivityData")
```

Comment:
dont think we need these anymore?  did we figure that out?

Reply:
Still need them on the backend. I can show you where if you want to think about how to eliminate them.